### ISSUE TITLE
Windows defender marked download as virus

### ISSUE THREAD

**Issue by Ohukainen** [aseprite/aseprite#4218]
I downloaded the .zip for [Aseprite v1.3.2](https://github.com/aseprite/aseprite/releases/tag/v1.3.2) from this [page](https://github.com/aseprite/aseprite/releases) and windows defender marked it as virus and deleted the file.

### Aseprite and System version

* Aseprite version: v1.3.2, download
* System: Windows 11

**Comment by loficat** [comment]
Same issue, version, source and system.  Both the zip file and a tmp file (?) were quarantined on both download attempts via Windows Defender.

Detected: Trojan:Script/Wacatac.B!ml

**Comment by CaseyHrt** [comment]
Same issue here, exactly as others have stated. 

Window Defender Details: "_This program is dangerous and executes commands from an attacker_"._

**Comment by dacap** [comment]
Hi everyone! I'm not sure what kind of heuristic Microsoft is using in this one, but the zip is just source code, there is no binary to execute inside (there are some Bash script files, but those are not executed in the regular compilation phase).

**Comment by UnnamedSteve** [comment]
![image](https://github.com/aseprite/aseprite/assets/145229894/a1e4d6eb-ffe0-4b14-81ea-1ae48e1323d2)

Same here!

**Comment by dacap** [comment]
This is a false positive of windows defender.

**Comment by CaseyHrt** [comment]
### A Quick Resolution for This Issue: 
#### To circumvent Windows Defender blocking the download of the Aseprite source file: 

1. Click Windows "Start" button
2. Search for and click "Windows Security"
3. Go to "Virus and Threat Protection", then "Virus and Threat Protection Settings"
4. Toggle "Real-Time Protection" to off
5. Download the latest Aseprite Source Code
6. **MAKE SURE** to reenable real-time protection if it is a safety measure you rely on!

**Comment by loficat** [comment]
As an aside for those who're interested, I was able to look at the file earlier.  After placing the archive in a sandbox, I extracted the files and ran several security apps to review them.  No issues were reported.  I'm not sure why it was picked up by Windows Defender, as it hadn't been flagged in the past.  Disclaimer:  This isn't a guarantee regarding the nature of the files or code.  Apologies for not posting sooner.  All the best.
